config/install/search_api.server.pantheon_solr8.yml

```diff
@@ -1,50 +0,0 @@
-langcode: en
```

Comment:
I'm worried about all of this config being removed and it's probably still referenced in the docs. Also: it's just "easier" for customers to have this as a starting point.

Is there any reason I'm missing on why you're removing this?

Reply:
Agreed. See slack conversation.